### PR TITLE
Add Graphical Abstract to README

### PR DESCRIPTION
**Pull request by rimangan**
# Description
Add graphical abstract to README

## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [ ] I performed a self-review of my code
- [ ] If it this a core feature, I have added thorough tests
- [ ] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 # gonomics
 A collection of genomics software tools written in Go (golang).
 
+![GonomicsReleaseFigure](https://github.com/vertgenlab/gonomics/assets/50707122/cf62dc98-1975-4d03-aabf-dcf4096def07)
```
Comment (craiglowe) [inline README.md:8]: I am having trouble telling where this file is stored.

**Review by craiglowe — COMMENTED** [review]
I think it is a cool idea to include this image as part of the readme, but I am not sure where it is stored.

File: README.md (hunk 1 above)
Comment (edotau) [reply README.md:8]: You can preview it how the README looks here: https://github.com/vertgenlab/gonomics/tree/rimangan-patch-1

**Review by edotau — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (craiglowe) [reply README.md:8]: That preview looks good, but I am wondering where the file is stored.  I don't see it being added as part of this PR.  Was it already added as part of a previous PR?

Comment (edotau) [reply README.md:8]: Looks like the image is uploaded to
- https://user-images.githubusercontent.com/50707122/253075286-cf62dc98-1975-4d03-aabf-dcf4096def07.png

I've only done this in PR comments and never added an actual image to a readme file, but it looks like if you edit the  readme file in the internet browser by clicking on the pencil, you can drag an image and it'll automatically upload to github's aws bucket. Here's a screen recording of my performing the action:

https://github.com/vertgenlab/gonomics/assets/11465271/5ade149a-db13-49bc-99a6-6a4748850b7a

**Review by cfauci — APPROVED** [review]
